store/tikv/client.go

```diff
@@ -68,24 +73,23 @@ func (c *rpcClient) SendReq(ctx goctx.Context, addr string, req *tikvrpc.Request
 	}
 	defer func() { sendReqHistogram.WithLabelValues(label).Observe(time.Since(start).Seconds()) }()
 
-	conn, err := c.p.GetConn(addr)
+	conn, err := c.p.Get(addr)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	connPoolHistogram.WithLabelValues("cop").Observe(time.Since(start).Seconds())
-	defer c.p.PutConn(conn)
-	resp, err := c.callRPC(ctx, conn, req)
+	connPoolHistogram.WithLabelValues(label).Observe(time.Since(start).Seconds())
+	client := tikvpb.NewTikvClient(conn)
```

Comment:
TikvClient is implemented to be just a simple wrapper of `ClientConn` in the kvproto generated code:
```
type tikvClient struct {
	cc *grpc.ClientConn
}
```
The expenses to construct a `tikvClient` cost really low. 
On the other hand, even with a constructed `tikvClient`, we still need to hold the instances of `ClientConn` so that we could call `Close()` in it.
So it's better to hold `ClientConn` instances directly in pool, and call `NewTikvClient` on the fly.